.github/workflows/deploy-release.yml

```diff
@@ -1,6 +1,33 @@
 name: Deploy Release Artifacts
 
 on:
```

Comment:
We can't have this here....as this is meant to be used from another workflow.  If these are here, then the real caller is not passed to this workflow